consensus/spos/bls/subroundStartRound.go

```diff
@@ -27,6 +28,7 @@ func NewSubroundStartRound(
 	extend func(subroundId int),
 	processingThresholdPercentage int,
 	executeStoredMessages func(),
```

Comment:
Maybe we can aggregate these function handlers arguments in an interface (for better code readability) somewhere in the future :)

Reply:
Yep, probably in the next task which will touch this part